website/docs/_posts/2017-10-05-configuration-options.md

```diff
@@ -511,14 +511,16 @@ Set this to true if you wish to receive one PR for every separate major version
 
 ## node
 
-Configuration specific for all node.js version updates (e.g. `.travis.yml`).
+Configuration specific for all NodeJS version updates (e.g. `engines` field in `package.json`).
 
 | name    | value                                         |
 | ------- | --------------------------------------------- |
 | type    | object                                        |
-| default | { groupName: 'node.js', lazyGrouping: false } |
+| default | { groupName: 'nodejs', lazyGrouping: false } |
```

Comment:
Changing the groupName like this will cause all existing PRs to be autoclosed and then recreated, because it will require a change in branchName. If kept as "Node.js" it would not have this impact.

Reply:
Ah! Well, then I definitely don't want to do that.